v22.1/follower-reads.md

```diff
@@ -1,197 +1,191 @@
+Use this function in an `AS OF SYSTEM TIME` statement as follows:
 
-For examples showing how to use these functions to get bounded staleness reads, see [Run queries that use bounded staleness follower reads](#run-queries-that-use-bounded-staleness-follower-reads).
+``` sql
+SELECT ... FROM ... AS OF SYSTEM TIME follower_read_timestamp();
+```
+#### Exact staleness follower reads demo
 
-### When to use bounded staleness reads
+The following video describes and demonstrates [exact staleness](#exact-staleness-reads) follower reads.
 
-Use [bounded staleness](#bounded-staleness-reads) follower reads when:
+<iframe width="560" height="315" src="https://www.youtube.com/embed/V--skgN_JMo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-- You need minimally stale reads from the nearest replica without blocking on conflicting transactions. This is possible because the historical timestamp is chosen dynamically and the least stale timestamp that can be served locally without blocking is used.
-- You can confine the read to a single statement that meets the [bounded staleness limitations](#bounded-staleness-read-limitations).
-- You need higher availability than is provided by exact staleness reads. Specifically, what we mean by availability in this context is:
-  - The ability to serve a read with low latency from a local replica rather than a leaseholder.
-  - The ability to serve reads from local replicas even in the presence of a network partition or other failure event that prevents the SQL gateway from communicating with the leaseholder. Once a replica begins serving follower reads at a timestamp, it will always continue to serve follower reads at that timestamp. Even if the replica becomes completely partitioned away from the rest of its range, it will continue to stay available for (increasingly) stale reads.
+#### Exact staleness follower reads in read-only transactions
 
-You should **not** use follower reads when your application cannot tolerate reading stale data, since the results of follower reads may not reflect the latest writes against the tables you are querying. However, for many applications, especially in [multi-region deployments](multiregion-overview.html), there are opportunities to do useful work with follower reads.
+You can set the [`AS OF SYSTEM TIME`](as-of-system-time.html) clause's value for all operations in a read-only transaction:
```

Comment:
"transactions" could be a link to https://www.cockroachlabs.com/docs/v21.2/transactions.html so folks can (re)familiarize themselves if they're going to be writing multi-stmt txns